.gitignore

```diff
@@ -23,3 +23,4 @@ archive.zip
 # vim temporary files
 *~
 .*.sw?
+.python-version
```

Comment:
This might be a requirement only for your local environment. Not sure it's necessary for the whole project.

Reply:
Agreed 🤓

Can you please remove it @andresbermeoq? :pray: